rust/etw_helpers/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+#[cfg(any(feature = "crossbeam"))]
+pub mod event_consumer_crossbeam;
+#[cfg(any(feature = "flume"))]
+pub mod event_consumer_flume;
+#[cfg(any(feature = "futures"))]
+mod event_consumer_futures;
+#[cfg(any(feature = "futures"))]
+mod event_consumer_futures_stream;
+//mod event_consumer_raw;
```

Comment:
This one worked without any dependencies on other crates, except it sometimes deadlocked and I just don't care enough to debug it when there are perfectly good implementations already.